packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Generated/Models/FloatExtensibleEnum.cs

```diff
@@ -0,0 +1,50 @@
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+using System.Globalization;
+
+namespace UnbrandedTypeSpec.Models
+{
+    public readonly partial struct FloatExtensibleEnum : IEquatable<FloatExtensibleEnum>
+    {
+        private readonly int _value;
+        private const int OneValue = 1;
+        private const int TwoValue = 2;
+        private const int FourValue = 4;
+
+        // Add Constructors
+        public FloatExtensibleEnum(int value)
+        {
+            _value = value;
+        }
+
+        public static FloatExtensibleEnum One { get; } = new FloatExtensibleEnum(OneValue);
+
+        public static FloatExtensibleEnum Two { get; } = new FloatExtensibleEnum(TwoValue);
+
+        public static FloatExtensibleEnum Four { get; } = new FloatExtensibleEnum(FourValue);
+
+        // Add Methods
+        public static bool operator ==(FloatExtensibleEnum left, FloatExtensibleEnum right) => left.Equals(right);
```

Comment:
Our code writer is not fully implemented to write xml documents for methods.
The description has been assigned to this `MethodSignature`, but the code writer just does not write it.

We could fix this later.